### PR TITLE
Implemented 5 tests for the server and database

### PR DESCRIPTION
**Pull request by mattman240**
https://trello.com/c/Vs36mudK/11-get-full-coverage-of-tests-for-code-l

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 2
```
Comment (ZeMunchkin) [inline .circleci/config.yml:5]: Are you sure this is the correct code for circleCI? Double check this

File: database-mongo/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+const itemSchema = mongoose.Schema({
+  id: { type: Number, unique: true },
+  pictures: String,
+  foodType: String,
+  name: String,
+  comment: String,
+});
+
+const Resturant = mongoose.model('resturant', itemSchema);
```
Comment (ZeMunchkin) [inline database-mongo/index.js]: not major, but restaurant*

File: database-mongo/index.test.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+// test for making sure each one has 3 pictures
+test('Should all have an array of 3 picture urls', (done) => {
+  select.selectAll((err, data) => {
+    const pictureArr = JSON.parse(data[0].pictures);
+    expect(pictureArr.length).toBe(3);
+    done();
+  });
+});
+// tests a select for a single resturant id
+test('Should all have an array of 3 picture urls', (done) => {
```
Comment (ZeMunchkin) [inline database-mongo/index.test.js]: This test is actually testing the same thing as the one on lines 11-15.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  "scripts": {
+    "react-dev": "webpack -d --watch",
+    "server-dev": "nodemon server/index.js",
+    "seed-data": "node database-mongo/index.js",
+    "test": "jest",
+    "coverage": "jest --coverage"
+  },
+  "author": "Matt Mullen",
+  "license": "ISC",
+  "devDependencies": {
```
Comment (ZeMunchkin) [inline package.json:15]: nearly all of your babel and webpack dependencies are going to be regular dependencies, not dev

File: server/index.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+const path = require('path');
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(express.static(path.join(__dirname, '/../react-client/dist')));
+
+app.get('/pictures', (req, res) => {
+  items.selectAll((err, data) => {
+    if (err) {
```
Comment (ZeMunchkin) [inline server/index.js:13]: could be shortened with a ternary operator. Matter of preference though

File: server/index.test.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+const request = require('supertest');
```
Comment (ZeMunchkin) [inline server/index.test.js:1]: Might be a good idea to make a directory for your tests that's separate from your other files. It'll make it easier for you when you start having more files to look through

**Review by ZeMunchkin — CHANGES_REQUESTED** [review]
Needs some minor modifications to be ready to merge

File: .circleci/config.yml (hunk 1 above)
Comment (mattman240) [reply .circleci/config.yml:5]: @ZeMunchkin fixed all the major things!

**Review by ZeMunchkin — APPROVED** [review]

**Pull request merged**